faker/providers/job/da_DK/__init__.py

```diff
@@ -0,0 +1,962 @@
+from .. import Provider as BaseProvider
+
+
+class Provider(BaseProvider):
+    """
+    Source: https://star.dk/it/saadan-arbejder-vi-med-it-i-styrelsen/oversigt-over-stillingsbetegnelser-og-kvalifikationer/
```

Comment:
This line is making the linter fail because it's too long. Can you mark it with `NOQA`?

```suggestion
    Source: https://star.dk/it/saadan-arbejder-vi-med-it-i-styrelsen/oversigt-over-stillingsbetegnelser-og-kvalifikationer/.  # NOQA
```